### PR TITLE
update MMM-JsonTable.js

### PR DESCRIPTION
**Pull request by late4marshmellow**
allowing nested arrays

### CODE REVIEW

**Comment by timdows** [comment]
Let me fix the build errors tomorrow

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # MMM-JsonTable
 
+All credit to @timdows her.
```
Comment (KristjanESPERANTO) [inline README.md:3]: Credits are good and would be appropriate for a fork. But here I don't think it should be listed.

File: README.md [hunk 2]
```diff
@@ -237,13 +240,95 @@ Configuration:
   position: 'top_right',
   header: 'HouseDB Kwh Statistics',
   config: {
-    url: 'https://xyz/abc/get.json',
+
+ url: 'https://xyz/abc/get.json',
```
Comment (KristjanESPERANTO) [inline README.md:244]: What for is this change?

File: README.md [hunk 3]
```diff
@@ -237,13 +240,95 @@ Configuration:
...
     arrayName: 'deviceKwhUsages',
     descriptiveRow: '<tr><td>Name</td><td>Today</td><td>ThisWeek</td><td>LastWeek</td><td>ThisMonth</td><td>LastMonth</td></tr>'
   }
 }
 ```
 
+Getting the data under "Forbrok"
+
+```
```
Comment (KristjanESPERANTO) [inline README.md:253]: Please add `json` after the backticks.

File: README.md (hunk 1 above)
Comment (late4marshmellow) [reply README.md:3]: Sorry never ment readme for PR

File: README.md (hunk 2 above)
Comment (late4marshmellow) [reply README.md:244]: Readme should not be pulled

**Comment by late4marshmellow** [comment]
Sorry @timdows never ment the readme to be pulled 😅
Can you ignore readme or should i make another request?

**Pull request closed without merging**